modules/installing-gcp-cluster-creation.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installing-gcp-cluster-creation_{context}"]
+= Configuring user-defined labels and tags for GCP
+
+After you successfully install an add-on service on your {product-title} cluster, you can access the service by using the GCP.
+
+.Prerequisites
+
+* You have successfully installed a service on your {product-title} cluster.
+
+* The installation program requires that a service account includes a `TagUser` role, so that the program can create the {product-title} cluster with defined tags at both organization and project levels.
+
+:FeatureName: Support for user-defined `labels` and `tags` for GCP
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+. Update the `.platform.gcp.userLabels` and `platform.gcp.userTags` fields to define the list of desired labels and tags in `install-config.yaml` file.
++
+[NOTE]
+====
+Labels and tags are defined during the `install-config.yaml` creation phase, and cannot be modified or updated with new labels and tags after cluster creation.
+====
+
+.Sample `install-config.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+featureSet: TechPreviewNoUpgrade
+platform:
+ gcp:
+   userLabels: <1>
+   - key: <label_key><2>
+     value: <label_value><3>
+   userTags: <4>
+   - parentID: <OrganizationID/ProjectID><5>
+     key: <tag_key_short_name>
+     value: <tag_value_short_name>
+----
+<1> Adds keys and values as labels to the resources created on GCP.
+<2> Defines the label name.
+<3> Defines the label content.
+<4> Adds keys and values as tags to the resources created on GCP.
+<5> The ID of the hierarchical resource where the tags are defined, at the Organization or the Project level.
+
+.The following are the requirements for the user-defined labels:
+
+* A label key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A label key and value must contain only lowercase letters, numeric characters, underscore (`_`), and dash (`-`).
+* A label key must start with a lowercase letter.
+* You can configure a maximum of 32 labels per resource. Each resource can have a maximum of 64 labels, and 32 labels are reserved for internal use by {product-title}.
+
+.The following are the requirements for the user-defined tags:
+
+* Tag key and tag value must already exist. {product-title} does not create the key and the value .
+* A tag `parentID` can be either `OrganizationID` or `ProjectID`:
+** An `OrganizationID` must consist of decimal numbers, and cannot have leading zeros.
+** A `ProjectID` must be 6 to 30 characters in length, can only contain lowercase letters, numbers, and hyphens, and must start with a letter, and cannot end with a hyphen.
```

Comment:
This can be broken down into multiple sentences.

Reply:
Ack.